src/PostForm.tsx

```diff
@@ -0,0 +1,104 @@
+import usersFromServer from './api/users';
```

Comment:
The `usersFromServer` data is directly imported here, which violates the requirement to prepare data in the `App` component and pass it down as props. Consider passing `users` as a prop to `PostForm`.